src/util/downloadUtils.ts

```diff
@@ -105,7 +105,7 @@ export async function downloadFromURL(
  */
 export async function hasStatusCode(
   url: URL,
-  statusCode: number
+  ...statusCodes: number[]
```

Comment:
I probably would have just done `statusCode: number | number[]` instead of varargs, but doesn't really matter.